EasyPost/BetaFeatures/Parameters/Shipments/Create.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using EasyPost.Utilities.Internal.Annotations;
+
+namespace EasyPost.BetaFeatures.Parameters.Shipments
+{
+    /// <summary>
+    ///     Parameters for <see cref="EasyPost.Services.ShipmentService.Create"/> API calls.
+    /// </summary>
+    public class Create : Parameters, IShipmentParameter
+    {
+        #region Request Parameters
+
+        [TopLevelRequestParameter(Necessity.Optional, "carbon_offset")]
+        public bool AddCarbonOffset { get; set; }
+
+        [TopLevelRequestParameter(Necessity.Optional, "shipment", "carrier_accounts")]
+        public List<EasyPost.Models.API.CarrierAccount>? CarrierAccounts { get; set; }
+
+        [TopLevelRequestParameter(Necessity.Optional, "shipment", "customs_info")]
+        public EasyPost.Models.API.CustomsInfo? CustomsInfo { get; set; }
+
+        [TopLevelRequestParameter(Necessity.Optional, "shipment", "insurance")]
+        public double Insurance { get; set; }
```

Comment:
```suggestion
        public double? Insurance { get; set; }
```

This is optional, let's make it nullable